js/wp-seo-featured-image.js

```diff
@@ -37,12 +43,11 @@ var thumbIdCounter = 0;
 function yst_overrideElemFunction() {
     if( document.getElementById( 'remove-post-thumbnail' ) != null ){
         thumbIdCounter = 0;
-        removeThumb = document.getElementById( 'remove-post-thumbnail' ).onclick;
```

Comment:
Ehm because it wasn't needed in this function and there was no comment saying it was needed for core functionality :) I'll restore it.